lxml-stubs/etree.pyi

```diff
@@ -577,7 +577,12 @@ def tostring(
     inclusive_ns_prefixes: Any = ...,
 ) -> _AnyStr: ...
 
-class _ErrorLog: ...
+class _LogEntry:
+    path: Optional[str]
+
+class _ErrorLog:
+    def __iter__(self) -> Iterator["_LogEntry"]: ...
```

Comment:
It's actually not `_ErrorLog` that's iterable but its base class. Also, again, there are lots of public attributes and methods missing here. It's best to add all of them.

Reply:
Thanks, I think I've added all the public attributes/properties/methods listed at https://lxml.de/apidoc/lxml.etree.html for `_LogEntry`, `_ErrorLog` and its parent classes. Let me know if I've missed some.